source/documentation/partials/_messagetypeintro.md

```diff
@@ -15,6 +15,45 @@ Message type details are as follows:
 - Links to applicable rules and conditions. 
 - Header elements (XML fields that can contain other XML fields) are highlighted in **bold**.
 
+## Message categorisation by sender
+
+This section lists the message types that are sent by NCTS and the message types that are sent by traders.
+
```

Comment:
Two minor issues:
- "This section" is a book/document reference that does not sit easily in a web page.
- "the NCTS" is better than "NCTS" (I'm judging this on cadence and the EU Transit Manual).

Suggestion:
Some message types are sent by the NCTS while other message types are sent by traders.